src/context.rs

```diff
@@ -702,7 +733,9 @@ impl Context {
     // REVIEW: Maybe more helpful to beginners to call this metadata_tuple?
     // REVIEW: Seems to be unassgned to anything
     // REVIEW: Should maybe make this take &[BasicValueEnum]?
+    #[llvm_versions(3.6..9.0)]
```

Comment:
Both this and the added version for 9 have the exact signature. Rather than having two definitions, we should have one and put the cfgs in the body itself. Like

```rust
#[llvm_versions(3.6..9.0)]
let metadata_value = unsafe { ... };
#[llvm_versions(9.0..=latest)]
let metadata_value = unsafe { ... };
```

And same for `metadata_string`